dashboard/app/controllers/api/v1/pd/application/teacher_applications_controller.rb

```diff
@@ -27,6 +27,7 @@ def principal_approval_not_required
 
     def on_successful_create
       @application.on_successful_create
+      @application.update_status_timestamp_change_log(current_user)
```

Comment:
This looks like it will log the applicant as the changing user. Do we want that? (asking because the test case in `application_serializer_test.rb` you didn't list a user for the initial status log entry)